packages/@aws-cdk/aws-elasticloadbalancingv2/lib/shared/base-load-balancer.ts

```diff
@@ -109,12 +109,9 @@ export abstract class BaseLoadBalancer extends Resource {
   public readonly loadBalancerSecurityGroups: string[];
 
   /**
-   * The VPC this load balancer has been created in, if available
-   *
-   * If the Load Balancer was imported, the VPC is not available.
+   * the vpc to create the load balancer inside
```

Comment:
You might want to leave this as `The VPC this load balancer has been created in`. These are the properties of the construct, not the input props.